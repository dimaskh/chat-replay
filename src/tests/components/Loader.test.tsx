// Libs
import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

// Tested components
import Loader, { spin } from 'components/Loader'

describe('components/Loader', () => {
  it('should render correctly', () => {
    const component = shallow(<Loader />)

    expect(shallowToJson(component)).toMatchSnapshot()
  })

  it('should define spin', () => {
    expect(spin).toBeDefined()
  })
})
