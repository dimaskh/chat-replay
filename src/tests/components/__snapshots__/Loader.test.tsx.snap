// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Loader should render correctly 1`] = `
<div
  className="sc-AxjAm emMRbY"
/>
`;
